lib/matplotlib/_api/__init__.pyi

```diff
@@ -42,7 +42,9 @@ def check_in_list(
     values: Sequence[Any], /, *, _print_supported_values: bool = ..., **kwargs: Any
 ) -> None: ...
 def check_shape(shape: tuple[int | None, ...], /, **kwargs: NDArray) -> None: ...
-def check_getitem(mapping: Mapping[Any, Any], /, **kwargs: Any) -> Any: ...
+def check_getitem(
+        mapping: Mapping[Any, _T], /, _error_cls: Type[Exception], **kwargs: Any
```

Comment:
```suggestion
        mapping: Mapping[Any, _T], /, _error_cls: type[Exception], **kwargs: Any
```

Reply:
I'm reluctant to spend time going back and forth on style here (and in general) - if folks want to enforce a consistent style, then it would be much more helpful to do it via an automatic formatter like `black` or `ruff`.

Reply:
👍  - I've opened https://github.com/matplotlib/matplotlib/pull/30312 to deal with cases like this automatically, and will fix this case in this PR.